speech_to_ra.py

```diff
@@ -0,0 +1,95 @@
+'''
+This file's purpose is to take a speech input from the user and convert it to Courier's Code.
+Input Type: Audio from Microphone
+Output Type: Text (Courier's Code Representation)
+
+For running this file:
+    - may need to 'brew install flac'
+    - may need to follow steps at:
+        - https://stackoverflow.com/questions/73268630/error-could-not-build-wheels-for-pyaudio-which-is-required-to-install-pyprojec
+
+All dependencies can be found in requirements.txt, which should be pip install -r'd before
+running locally.
+
+Your machine needs to be connected to the internet in order to run recognize_google().
+'''
+
+from re import M
+import sys
+import subprocess
+from subprocess import STDOUT, check_call
+import os
+import speech_recognition as sr
+import pyttsx3
+
+couriers_code = {
```

Comment:
Constants in Python are usually written in all caps
```suggestion
COURIERS_CODE = {
```